file_organizer_v2/examples/demo_comparison_viewer.py

```
@@ -0,0 +1,167 @@
+
+# Add src to path
+sys.path.insert(0, str(Path(__file__).parent.parent / "src"))
+
+from file_organizer.services.deduplication.viewer import ComparisonViewer
+from rich.console import Console
+
+
+def demo_single_comparison():
+    """Demo showing comparison of a few duplicate images."""
+    console = Console()
+    viewer = ComparisonViewer(console=console)
```

Ruff (0.14.13)

Local variable `viewer` is assigned to but never used

Remove assignment to unused variable `viewer`

(F841)